fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/FlatMapPipelinedCursor.java

```diff
@@ -267,6 +267,7 @@ private synchronized void addEntryToPipeline(PipelineQueueEntry pipelineQueueEnt
         if (closed) {
             pipelineQueueEntry.close();
         }
+        pipelineQueueEntry.getNextInnerPipelineFuture();
```

Comment:
Do we need both this change and the change in the `PipelineQueueEntry` constructor? If I'm reasoning about this correctly, it seems like we only need one of the two (that this is extraneous if the next future is fired off during the constructor, and the constructor call is extraneous if the future is created here)

Reply:
Correct, this was an oversight, let me remove that.